src/EnergyPlus/GeneralRoutines.cc

```diff
@@ -297,13 +295,11 @@ void ControlCompOutput(EnergyPlusData &state,
                     ZoneController.CalculatedSetPoint = ZoneInterHalf.MaxFlow; // CR7253
                 }
                 // Set the Actuated node MassFlowRate with zero value
-                if (present(LoopNum)) { // this is a plant component
+                if (plantLoc.loopNum) { // this is a plant component
```

Comment:
I assumed if plantLoc.loopNum is not present, it would be zero here since loopNum is initialized with value zero.